src/plugins/imagery/components/ImageControls.vue

```diff
@@ -26,11 +26,20 @@
     role="toolbar"
     aria-label="Image controls"
   >
-    <imagery-view-menu-switcher :icon-class="'icon-brightness'" :title="'Brightness and contrast'">
+    <imagery-view-menu-switcher
+      :icon-class="'icon-brightness'"
+      :aria-label="'Brightness and contrast'"
+      :title="'Brightness and contrast'"
+    >
       <filter-settings @filter-changed="updateFilterValues" />
     </imagery-view-menu-switcher>
 
-    <imagery-view-menu-switcher v-if="layers.length" :icon-class="'icon-layers'" :title="'Layers'">
+    <imagery-view-menu-switcher
+      v-if="layers.length"
+      :icon-class="'icon-layers'"
+      :aria-label="'Layers'"
+      :title="'Layers'"
```

Comment:
These shouldn't be `v-binds` if they're static.
```suggestion
      icon-class="icon-layers"
      aria-label="Layers"
      title="Layers"
```

We should fix all instances of this in this file. `:` is short for `v-bind:`, meaning that the prop is reactive and bound to a data property in the parent.